developer/openshift/gitops/argocd/pipeline-service-storage/postgres.yaml

```diff
@@ -14,12 +14,12 @@ spec:
   source:
     chart: postgresql
     repoURL: https://charts.bitnami.com/bitnami
-    targetRevision: 12.2.7
+    targetRevision: 14.0.5
     helm:
       releaseName: postgres
       parameters:
         - name: image.tag
-          value: 13.10.0
+          value: 13.14.0
```

Comment:
While I understand this change is for dev only, just an FYI : In our prod clusters we use the `Postgres version 13.11`. 
Btw I saw your comment @avinal 

> I went through the bitnami chart index, this is the only one in postgres 13.x.x that is updated. All others are either deprecated or removed.

Reply:
We will need to change that as well if they are using the same config.